tests/test_url_config.py

```diff
@@ -1,86 +1,125 @@
-import json

```

Comment:
can you fix this messed up github history of this entire file? It's quite annoying to have to resolve merge conflicts later

Reply:
i can try...

Reply:
hmm i tried reordering to minimize the changed rows but i cant git diff to identify only the changed regions independently. any ideas?

Reply:
sometimes it's the end character that messes these things up, but if I'm not able to find that, I usually think re-making the PR saves me the time from going over all of the lines.

Is it your IDE that does something funny or so?

Reply:
Indeed when i switched the line endings to CLRF it fixed it. Not sure how windows line endings made it into the file but ok.